src/main/java/no/sikt/nva/scrapers/EmbargoScraper.java

```diff
@@ -47,16 +47,23 @@ private static boolean fileIsEmpty(File file) {
     }
 
     private static Map<String, List<Embargo>> convertStringToEmbargoObjects(String contentFileAsString) {
-        var listOfStringObjects = Arrays.asList(
-            contentFileAsString.replaceAll(EMPTY_LINE_REGEX, StringUtils.EMPTY_STRING)
-                .replace("|", ";")
-                .split("\n"));
-        var embargoes = listOfStringObjects.stream()
+        var listOfStringObjects = Arrays.asList(contentFileAsString.replaceAll(EMPTY_LINE_REGEX, StringUtils.EMPTY_STRING)
+                                                                    .replace("|", ";")
+                                                                    .split("\n"));
+        var embargoes = removeIgnoredLines(listOfStringObjects).stream()
                             .map(EmbargoScraper::convertToEmbargo)
                             .collect(Collectors.toList());
         return createEmbargoHandleMap(embargoes);
     }
 
+    private static List<String> removeIgnoredLines(List<String> listOfStringObjects) {
+        var list = new ArrayList<>(listOfStringObjects);
+        list.remove(0);
+        list.remove(0);
+        list.remove(list.size() - 1);
+        return list;
+    }
```

Comment:
I think this is fragile. 
If the "brage-migrators" stick to their old routine, we are throwing out more than we want to.
Can we have a check if they should be removed or not, before removing these lines?